core/templates/dev/head/domain/exploration/FallbackObjectFactory.js

```diff
@@ -17,7 +17,9 @@
  * domain objects.
  */
 
-oppia.factory('FallbackObjectFactory', [function() {
+oppia.factory('FallbackObjectFactory', [
+    'OutcomeObjectFactory',
+  function(OutcomeObjectFactory) {
   var Fallback = function(trigger, outcome) {
```

Comment:
But indent this line and the following by 2, and split up the }]) at the end into } and ]) to match the opening brackets.